PrincessBrideTrivia/PrincessBrideTrivia/Program.cs

```diff
@@ -21,9 +21,12 @@ public static void Main(string[] args)
 
     public static string GetPercentCorrect(int numberCorrectAnswers, int numberOfQuestions)
     {
-        return (numberCorrectAnswers / numberOfQuestions * 100) + "%";
+        //return (numberCorrectAnswers / numberOfQuestions * 100) + "%"; This Line returns a int and will be 0% if they miss one question
+        double result = (int)(((double)numberCorrectAnswers / (double)numberOfQuestions) * 100);
+        return result + "%";
```

Comment:
```suggestion
        return ((float)numberCorrectAnswers / numberOfQuestions) * 100);
```
Just to simplify, and centralize all that @Joshua-Pedersen mentioned this is all you should need.
1. The numbers coming into these methods are already int's so the cast is unnecessary
2. The cast on both numbers is unnecessary
3. The return can just be combined into one line. Not wrong, just unnecessary code.